packages/agoric-cli/src/chain-config.js

```diff
@@ -5,6 +5,8 @@ export const CENTRAL_DENOM = 'urun';
 export const MINT_DENOM = 'ubld';
 export const STAKING_DENOM = 'ubld';
 export const STAKING_MAX_VALIDATORS = 150;
+export const COSMOS_TRANSFER_PORT_ID = 'cosmos-transfer';
```

Comment:
I'm still fuzzy on `transfer` vs. `cosmos-transfer`. Is it worth adding a comment about that here?